src/content/en/fundamentals/progressive-web-apps/prpl.md

```diff
@@ -0,0 +1,243 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: 
+
+{# wf_updated_on: 2016-09-28 #}
+{# wf_published_on: 2016-09-28 #}
+
+# The PRPL Pattern {: .page-title }
+
+{% include "web/_shared/contributors/addyosmani.html" %}
+
+Dogfood: PRPL is a pattern we believe makes sense for applications that can be
```

Comment:
I'm ok with using Dogfood: for experimental stuff. I know it's not typical, but it meets the bar to me. Happy to discuss when we meet next week.